containers/sprm-to-json/context/main.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+from glob import glob
+from pathlib import Path
+from os import makedirs
+import json
+
+from shapely.geometry import Polygon
+import pandas as pd
+import numpy as np
+
+# Number of vertices for the polygon boundary data.
+NUM_VERTICES = 20
+GENES_FILE_SUFFIX = ".ome.tiff-cell_channel_mean.csv"
+CLUSTER_FILE_SUFFIX = ".ome.tiff-cell_cluster.csv"
+POLYGON_FILE_SUFFUX = ".ome.tiff-cell_polygons_spatial.csv"
+
+
+def sprm_to_items(input_file):
+    df = pd.read_csv(input_file)
+    df = df.set_index("ID")
+    df_items = df.T.to_dict().items()
+    return df_items
+
+
+def write_genes_or_factors_to_cells(input_file, cells={}, genes=True):
+    vitessce_key = "genes" if genes else "factors"
+    df_items = sprm_to_items(input_file)
+    for (cell_key, value) in df_items:
+        cells[cell_key][vitessce_key] = {}
+        for value_key in value:
+            cells[cell_key][vitessce_key][value_key] = (
+                value[value_key] if genes else str(value[value_key])
+            )
+    return cells
+
+
+def write_polyon_bounds(input_file, cells):
+    df_items = sprm_to_items(input_file)
+    for (k, v) in df_items:
+        shape = eval(v["Shape"])
```

Comment:
eval worries me. They are really giving us python data structures? If not JSON, could [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval) be used?